packages/TimePicker/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@paprika/time-picker",
+  "version": "0.0.0",
```

Comment:
is this correct?

Reply:
yes lerna will fix it during the publishing phase.